examples/cpp/security/hello_world_profile.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<profiles xmlns="http://www.eprosima.com" >
+    <participant profile_name="secure_hello_world_participant_profile" is_default_profile="true">
+        <domainId>0</domainId>
+        <rtps>
+            <name>hello_world_participant</name>
+            <propertiesPolicy>
+                <!-- Set authentication for DDS Domain communication  -->
+                <properties>
+                    <!-- Activate DDS:Auth:PKI-DH plugin -->
+                    <property>
+                        <name>dds.sec.auth.plugin</name>
+                        <value>builtin.PKI-DH</value>
+                    </property>
+                    <!-- Configure DDS:Auth:PKI-DH plugin -->
+                    <property>
+                        <name>dds.sec.auth.builtin.PKI-DH.identity_ca</name>
+                        <value>file://maincacert.pem</value>
```

Comment:
I have not tested the example yet but I would say that this path should be `certs/maincacert.pem`, same applies for the rest of paths